lib/plan_executor/executor.rb

```diff
@@ -160,19 +160,8 @@ def wait_until(timeout, retry_interval)
       end
     end
 
-    # Plan context doesn't make sense for most transports but it is tightly
-    # coupled with the orchestrator transport since the transport behaves
-    # differently when a plan is running. In order to limit how much this
-    # pollutes the transport API we only handle the orchestrator transport here.
-    # Since we callt this function without resolving targets this will result
-    # in the orchestrator transport always being initialized during plan runs.
-    # For now that's ok.
-    #
-    # In the future if other transports need this or if we want a plan stack
-    # we'll need to refactor.
-    def start_plan(plan_context)
-      @transport.plan_context = plan_context
-      @plan_logging = true
+    def conn_plan_job(plan_id)
```

Comment:
Rubocop errored on 'set_plan_job', and since this isn't setting a var `plan_job=` also isn't what gets called with `executor.plan_job`. This was my compromise. I'll take another stab at using either of the other 2, or just turning off rubocops